.changeset/rude-mice-study.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+fix: Ignore OPTIONS requests in Wrangler's oauth server
```

Comment:
I know this is a backport of the other changeset, but as I commented there, I think it would be helpful to include information about symptoms of the problem that the user would have seen and the fact that this was needed to fix a change in Chrome, mentioning the version etc, so that it makes it easier for users to identify this release as solving that problem.

Reply:
It might be worth just calling out the error messages that people might see:

``
Expired! Auth code or refresh token needs to be renewed.
```

and

```
ErrorInvalidGrant
```